src/app/clusters/door-lock-server/door-lock-server-schedule.c

```diff
@@ -156,7 +156,7 @@ bool emberAfDoorLockClusterSetWeekdayScheduleCallback(uint8_t scheduleId, uint16
     if (rfProgrammingEventMask & BIT(0))
     {
         emberAfFillCommandDoorLockClusterProgrammingEventNotification(0x01, 0x00, userId, &userPin, 0x00, 0x00, 0x00, &userPin);
-        SEND_COMMAND_UNICAST_TO_BINDINGS();
+        // SEND_COMMAND_UNICAST_TO_BINDINGS();
```

Comment:
If the cli API is removed can't we simply remove those calls and remove the declaration at https://github.com/project-chip/connectedhomeip/blob/74798d4d23b1ca50177b5802fd0e3cb13b0d1a92/src/app/util/af.h#L1382 ?

Reply:
@vivien-apple I don't follow. What does the cli API have to do with this?  `emberAfDoorLockClusterSetWeekdayScheduleCallback` is called by command dispatch when the "Set Week Day Schedule" command (ZCL v8 section 7.3.2.15.13) is received.  And the "RF Programming Event" bits are part of the ZCL spec too, as far as I can tell

Reply:
My bad. I have misread the documentation... replacing ``` client command API```by ```CLI API```in my head.